src/components/select/select.tsx

```diff
@@ -77,12 +81,14 @@ export class Select implements FormComponent {
 
   private getChildOptions() {
     return Array.from(this.element.querySelectorAll("gx-select-option")).map(
-      (option: any) => ({
-        disabled: option.getAttribute("disabled") !== null,
-        innerText: option.innerText,
-        selected: option.getAttribute("selected") !== null,
-        value: option.value
-      })
+      (option: any) => {
+        return {
+          disabled: option.getAttribute("disabled") === "true" ? true : false,
+          innerText: option.innerText,
+          selected: option.getAttribute("selected") === "true" ? true : false,
```

Comment:
Maybe this is a good opportunity to use the properties instead of the attributes:
```suggestion
          disabled: option.disabled,
          innerText: option.innerText,
          selected: option.selected,
```

Reply:
That mapping has been created by you. There, you do a `querySelectorAll` to get all children. And those children are HTMLElements. I cannot access to those properties like a component...
Btw, I will try to do that!

Reply:
Yes, I remember I did, but everything can be improved 😄

Reply:
I realized you can access it! What weirdo...
Changes applied in bf88e1d